### PR TITLE
Minor: Rename mentions of Apple Business Chat to Apple Messages for Business RD-21568

### PR DESCRIPTION
**Pull request by kalilz4485**
- Replaced mentions of "Apple Business Chat" with "Apple Messages for Business"

### CODE REVIEW

File: docs/interactions/structured-messages/time-select.md [hunk 1]
```diff
@@ -66,8 +66,8 @@ curl -X POST "https://[YOUR DOMAIN].api.digital.ringcentral.com/1.0/contents"
 | **`structured_content.location.radius`** | Integer | The radius around latitude/longitude of the location. |
 | **`structured_content.location.title`** | String | The title of the location. |
 
-## Example: Apple Business Chat (Time Picker)
+## Example: Apple Messages for Business (Time Picker)
 
-Nothing specifically unique as this is an Apple Business Chat specific structured message type.
+Nothing specifically unique as this is an Apple Messages for Business specific structured message type.
```
Comment (kalilz4485) [inline docs/interactions/structured-messages/time-select.md:71]: appletime

File: docs/interactions/structured-messages/select.md [hunk 2]
```diff
@@ -47,9 +47,9 @@ curl -X POST "https://[YOUR DOMAIN].api.digital.ringcentral.com/1.0/contents"
 | **`structured_content.items.title`** | String | The title of the item. Limited to 80 characters. |
 | **`structured_content.items.payload`** | String | **Optional**. Payload that will be returned with the structured response. Limited to 1000 characters. |
 
-## Example: Apple Business Chat (Quick Replies)
+## Example: Apple Messages for Business (Quick Replies)
 
-The following example uses Apple Business Chat with quick replies.
+The following example uses Apple Messages for Business with quick replies.
```
Comment (kalilz4485) [inline docs/interactions/structured-messages/select.md:52]: applequick

File: docs/interactions/structured-messages/list.md [hunk 3]
```diff
@@ -54,7 +54,7 @@ curl -X POST "https://[YOUR DOMAIN].api.digital.ringcentral.com/1.0/contents"
 | **`structured_content.items.subtitle`** | String | **Optional**. The subtitle of the item. Limited to 100 characters. |
 | **`structured_content.items.section_identifier`** | String | **Optional if there's no section**. The identifier of the section where the item is.<br>If there's no section, the section_identifier field should be removed.<br>Each section must have at least 1 item.<br>Limited to 200 characters. |
 
-## Example: Apple Business Chat (List Picker)
+## Example: Apple Messages for Business (List Picker)
```
Comment (kalilz4485) [inline docs/interactions/structured-messages/list.md:57]: applelist

File: docs/interactions/structured-messages/index.md [hunk 4]
```diff
@@ -56,12 +56,12 @@ This section describes the generic compatibility logic of the API. Check the fol
...
+| Non-standard attachment type | If the channel does not support the attachment, we provide an attachment_fallback_id field to be used instead.<br><br>The attachment_fallback_id is validated **if the attachment_id is present**. The validations are:<ul><li>Must be public</li><li>Must be less than 5MB</li><li>Must be a jpg, jpeg or png</li></ul>For example, Apple Messages for Business supports mp4 as its rich_link attachment. But when sending a payload with a mp4 attachment on Engage Messaging , the attachment_fallback_id field will be used if present, or return an error. |
 | Structured content type not existing in the channel | If a structured content type does not exist in a channel, the API will try to best represent it by converting it to another type the channel supports.<br><br>See the channel specificities section for a more detailed behavior on this conversion.<br><br>For example, the rich link type does not exist in Google Business Messages. So an Engage Digital Rich Link is converted to a Google Business Messages Rich Card with a single button representing the url. |
 
 ## Channel Specificities
 
-### Apple Business Chat
+### Apple Messages for Business
```
Comment (kalilz4485) [inline docs/interactions/structured-messages/index.md:64]: apppleoverview3

File: docs/interactions/structured-messages/index.md [hunk 5]
```diff
@@ -56,12 +56,12 @@ This section describes the generic compatibility logic of the API. Check the fol
...
-| Non-standard attachment type | If the channel does not support the attachment, we provide an attachment_fallback_id field to be used instead.<br><br>The attachment_fallback_id is validated **if the attachment_id is present**. The validations are:<ul><li>Must be public</li><li>Must be less than 5MB</li><li>Must be a jpg, jpeg or png</li></ul>For example, Apple Business Chat supports mp4 as its rich_link attachment. But when sending a payload with a mp4 attachment on Engage Messaging , the attachment_fallback_id field will be used if present, or return an error. |
+| Non-standard attachment type | If the channel does not support the attachment, we provide an attachment_fallback_id field to be used instead.<br><br>The attachment_fallback_id is validated **if the attachment_id is present**. The validations are:<ul><li>Must be public</li><li>Must be less than 5MB</li><li>Must be a jpg, jpeg or png</li></ul>For example, Apple Messages for Business supports mp4 as its rich_link attachment. But when sending a payload with a mp4 attachment on Engage Messaging , the attachment_fallback_id field will be used if present, or return an error. |
```
Comment (kalilz4485) [inline docs/interactions/structured-messages/index.md:59]: appleoverview2

File: docs/interactions/structured-messages/index.md [hunk 6]
```diff
@@ -31,7 +31,7 @@ Engage Digital provides an omnichannel API to create structured messages. Since
 
   |  | **select** | **rich_link** | **template** | **carousel** | **list** | **time_select** | **apple_pay** | **authentication** |
   |-|-|-|-|-|-|-|-|-|
-  | **Apple Business Chat** | **<span style="color:green">Yes</span>** | **<span style="color:green">Yes</span>** | N/A | N/A | **<span style="color:green">Yes</span>** | **<span style="color:green">Yes</span>** | **<span style="color:green">Yes</span>** | **<span style="color:green">Yes</span>** |
+  | **Apple Messages for Business** | **<span style="color:green">Yes</span>** | **<span style="color:green">Yes</span>** | N/A | N/A | **<span style="color:green">Yes</span>** | **<span style="color:green">Yes</span>** | **<span style="color:green">Yes</span>** | **<span style="color:green">Yes</span>** |
```
Comment (kalilz4485) [inline docs/interactions/structured-messages/index.md:34]: applestrucoverview

File: docs/interactions/structured-messages/apple-pay.md [hunk 7]
```diff
@@ -51,9 +51,9 @@ curl -X POST "https://[YOUR DOMAIN].api.digital.ringcentral.com/1.0/contents"
 | **`structured_content`** | Object | Payload of the structured message. |
 | **Structured Content Settings** | | |
 | **`structured_content.type`** | String | Type of the structured message. Must be "apple_pay". |
-| **`structured_content.payment_gateway_url`** | String | The url of the payment gateway matching the one configured in your business chat account. |
+| **`structured_content.payment_gateway_url`** | String | The url of the payment gateway matching the one configured in your Messages for Business account. |
 | **`structured_content.merchant_session`** | Object | The complete payload of the merchant session request from Apple. |
-| **`structured_content.merchant_identifier`** | String | The merchant identifier configured in your business chat account. |
+| **`structured_content.merchant_identifier`** | String | The merchant identifier configured in your Messages for Business account. |
```
Comment (kalilz4485) [inline docs/interactions/structured-messages/apple-pay.md:56]: applepaysettings

File: docs/interactions/structured-messages/apple-auth.md [hunk 8]
```diff
@@ -88,8 +88,8 @@ The customer will receive a message including the body and a button that will op
 }
 ```
 
-## Example: Apple Business Chat (Apple Pay)
+## Example: Apple Messages for Business (Apple Pay)
 
-Nothing specifically unique as this is an Apple Business Chat specific structured message type.
+Nothing specifically unique as this is an Apple Messages for Business specific structured message type.
```
Comment (kalilz4485) [inline docs/interactions/structured-messages/apple-auth.md:93]: apple1

File: docs/interactions/structured-messages/apple-auth.md [hunk 9]
```diff
@@ -3,7 +3,7 @@
 This structured message provides a way to send an OAuth2 authentication request to the customer. See [Channel capabilities](../#channel-capabilities) to know on which channel you can use this structured message.
 
 ## Prerequisites
-* Set up OAuth URL and Token URL in "End User Authentication” on your Business Chat Account.
+* Set up OAuth URL and Token URL in "End User Authentication” on your Messages for Business Account.
```
Comment (kalilz4485) [inline docs/interactions/structured-messages/apple-auth.md:6]: apple2

File: docs/interactions/structured-messages/apple-auth.md (hunk 8 above)
Comment (kalilz4485) [reply docs/interactions/structured-messages/apple-auth.md:93]: btw on this one I replaced "Apple Pay" with "Apple Authentication", was probably a copy paste error from a long time ago

File: docs/interactions/structured-messages/index.md [hunk 10]
```diff
@@ -56,12 +56,12 @@ This section describes the generic compatibility logic of the API. Check the fol
...
 | Structured content type not existing in the channel | If a structured content type does not exist in a channel, the API will try to best represent it by converting it to another type the channel supports.<br><br>See the channel specificities section for a more detailed behavior on this conversion.<br><br>For example, the rich link type does not exist in Google Business Messages. So an Engage Digital Rich Link is converted to a Google Business Messages Rich Card with a single button representing the url. |
 
 ## Channel Specificities
 
-### Apple Business Chat
+### Apple Messages for Business
 
 * [List Picker (select)](../structured-messages/select/#example-apple-business-chat-list-picker)
```
Comment (ylecuyer) [inline docs/interactions/structured-messages/index.md:66]: imo also those links need to be fixed

Comment (kalilz4485) [reply docs/interactions/structured-messages/index.md]: Thanks, fixed

**Pull request merged**